client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/Util.java

```diff
@@ -124,4 +124,16 @@ public static WrappedByteArray wrapBytes(byte[] cacheName) {
       }
       return wrappedCacheName;
    }
+
+   public static int getIntProperty(String name, int defaultValue) {
```

Comment:
suggestion, move this to `commons` to avoid duplicated code in "new hot rod client module"